integration/README.md

```diff
@@ -25,7 +25,7 @@ virtual environment and install required dependencies:
 ```shell
 $ python3 -m venv .venv
 $ source .venv/bin/activate
-$ pip install --requirements bin/requirements-fozen.txt
```

Comment:
Ok for the typo on frozen but we try to use long option as muxh of possible so it's better to keep `--requirements` instead of `-r`

Reply:
Ah there is also a typo: it's `--requirement` without s

Reply:
No problem :)